Library/Homebrew/cmd/install.rb

```diff
@@ -325,8 +325,8 @@ def install_formula(f)
     fi.build_bottle         = args.build_bottle?
     fi.interactive          = args.interactive?
     fi.git                  = args.git?
-    fi.fetch
     fi.prelude
+    fi.fetch
```

Comment:
This is inconsistent with other `FormulaInstaller` initialisations and means that the formula will be locked while dependencies are fetched (we we want to avoid). I'd suggest the disabled/deprecated checks either be duplicated in `fetch` or moved to `prelude`.